addon/mixins/route.js

```diff
@@ -217,6 +217,10 @@ export default Ember.Mixin.create({
     return false;
   },
 
+  updateModel(model, infinityModel) {
```

Comment:
Actually, I'm wondering if we should rename this method to something more specific to `ember-infinity`? It's conceivable that an app could already have a route method called `updateModel` that's unrelated to this addon.